_posts/2024-07-22-pyopensci-at-scipy-2024.md

```diff
@@ -0,0 +1,463 @@
+making packaging decisions. People who have reviewed our packaging content
+include both those in the scientific Python ecosystem and those working on core
+Python challenges. These people include:
+
+* Python Packaging Authority members (PyPA)
+* Packaging tools maintainers
+* Packaging experts
+* Scientists
+* People new to packaging
+
+All of these people have helped pyOpenSci create a guidebook that is both
+accurate and beginner-friendly.
+
+<figure>
+    <a href="{{ site.baseurl }}/images/blog/2024/july/pyopensci-package-guide-review.png"><img src="{{ site.baseurl }}/images/blog/2024/july/pyopensci-package-guide-review.png" alt="Image of the pyOpenSci package guide review process. There are three main boxes on the slide labeled 'expert feedback,' 'community review,' and 'sprints and bug bashes.' The top of the image is titled 'The Power of Community Review.'"></a>
+    <figcaption>pyOpenSci engages people from all parts of the Python community, with all levels of expertise, in our packaging guide review process. This engagement ensures a high-quality and beginner-friendly guidebook.</figcaption>
+</figure>
+
+One of the decisions we made as a community early on was to use Hatch as an
+end-to-end packaging tool. We believe Hatch is user-friendly and supports many
+of the use cases scientists have when sharing their code.
+
+<figure>
+    <a href="{{ site.baseurl }}/images/blog/2024/july/pyopensci-packaging-hatch.png"><img src="{{ site.baseurl }}/images/blog/2024/july/pyopensci-packaging-hatch.png" alt="Image titled 'Creating an Opinionated Tutorial.' Below it lists four bullet points about Hatch: 1. Responsive maintainer, 2. User friendly, 3. Adheres to standards, 4. Doesn't lock you in." style="width: 80%;"></a>
+    <figcaption>pyOpenSci is teaching with Hatch now because we have had good success with using Hatch and engaging with its forward-thinking maintainer.</figcaption>
+</figure>
+
+I met some great people and had good discussions about peer review and Python
+packaging. The presentation will be on YouTube at some point, and I will update
+this post with the link when it's live. In the meantime, my slides are available
+on [Zenodo pyOpenSci community](https://zenodo.org/records/8045448).
+
+## The hallway track
+
+It seems like this year has been the year of the hallway track. The hallway
+track refers to the time spent in the hallways of a meeting, talking to people
+you don't normally get to work with in person. Sometimes, the hallway track can
+be even more valuable than attending the talks. You can always watch the talks
+later on YouTube, but you can't always collaborate in real life like you can at
+a meeting like SciPy, which attracts people from around the world.
+
+I spent a lot of time talking with colleagues, friends, and community members
+about all things Python, open source, and open science.
+
+* I worked with Sarah Kaiser on our new GitHub container to support workshops.
```

Comment:
is there something we could link to for the GitHub container?

Reply:
oh yes - it's not merged yet but i could link to the PR! i'll do that locally.